libraries/AP_Scripting/docs/docs.lua

```diff
@@ -517,10 +517,20 @@ function motor_factor_table_ud:roll(index, value) end
 local SocketAPM_ud = {}
 
 -- Get a new socket
----@param datagram boolean
+---@param datagram integer -- set to 1 for UDP, 0 for TCP
 ---@return SocketAPM_ud
 function Socket(datagram) end
 
+-- return an IPv4 address given a string
+---@param str_address string -- ipv4 address as string
+---@return uint32_t_ud -- ipv4 address
+function string_to_ipv4_addr(str_address) end
+
+-- return a string representation of ipv4 address
+---@param uint32_t_ud addr -- ipv4 address
```

Comment:
```suggestion
---@param addr uint32_t_ud|integer|number -- ipv4 address
```
name and type were swapped. And the magic casting thing.

Reply:
changed